### ISSUE TITLE
`get lags` throws "non-zero number of bytes remaining"

### ISSUE THREAD

**Issue by royaldark** [segmentio/topicctl#15]
First off, thank you for releasing this amazing tool! It has simplified my life a lot :) 

Almost everything seems to work great, except the `get lags` command always seems to throw an error.

Example REPL session:

```
>>> get groups
> Groups:
------------------+--------------
       GROUP      | COORDINATOR
------------------+--------------
  1               | 1
  my-group-1      | 1
  my-group-2      | 1
  schema-registry | 2
------------------+--------------
>>> get lags topic1 my-group-1
[2020-08-27 13:30:53] ERROR Error: Got non-zero number of bytes remaining: 1993
>>> get lags topic2 my-group-2
[2020-08-27 13:31:16] ERROR Error: Got non-zero number of bytes remaining: 4
```

The `1` consumer group also seems suspicious: if I run the built-in `kafka-consumer-groups.sh` command, I only see two groups: `my-group-1` and `my-group-2`.

All other commands I've tried work just fine (`get topics`, `get brokers`, etc)

**Comment by yolken-segment** [comment]
Interesting, have not seen that behavior before on our side. A few follow-up questions:

1. Does `get offsets` work correctly for those topics?
2. What kafka version are you running?
3. Which library or libraries did you use to create the consumers associated with those groups?

Thanks for the report!

**Comment by erikdw** [comment]
I've reproduced what seems to be this same issue (using Kafka v2.4.0+ and `kafka-console-consumer.sh`). 
The cause seems to be [decoding of the Member Metadata field](https://github.com/segmentio/kafka-go/blob/711387657efffcd9af516d7853cebcdf39688d57/describegroups.go#L319-L355).  We realized that the binary content of this field can vary per-consumer type, and thus we cannot just assume our existing decoding logic would work for all cases.

@yolken-segment and I discussed this, and I'll be making a change to `kafka-go` to allow the decoding of the Member Metadata to fail and then a change to `topicctl` to allow for the Member Metadata being empty.
